doc/source/conf.py

```diff
@@ -93,21 +84,119 @@
 # The master toctree document.
 master_doc = "index"
 
-# customised model for cover page
-latex_maketitle = r"""
-\begin{titlepage}
-\sphinxlogo
-\custitle
-\end{titlepage}
-"""
-# additional style file for coverpage
-latex_additional_files = ["cover.sty", ansys_logo]
-
-# change the preamble of latex with latex_maketitle
+# change the preamble of latex with customized title page
 latex_elements = {
     "sphinxsetup": "",
     "passoptionstopackages": r"""
     \PassOptionsToPackage{table}{xcolor}""",
-    "preamble": r"""\usepackage{cover}""",
-    "maketitle": latex_maketitle,
+    "preamble": r"""\renewcommand{\sphinxmaketitle}{%
```

Comment:
Here starts the title page customization.

Reply:
We should try to avoid having all this code in the `conf.py

Ansys and PyAnsys projects are expected to benefit from this PDF cover. Thus, we need to make things as easy as possible for users.

We need to explore how to provide this LaTeX config by default with the PyAnsys Sphinx Theme. Notice that these modifications were made to the `conf.py` file. This means that they will not be shipped with the theme.

It is likely that we have to play with the `__init__.py` file for exposing this to any pyansys-sphinx-theme user.

Reply:
Add this as a extra file in the library, and then you force "conf.py" to read it??